odoo/http.py

```diff
@@ -6,7 +6,6 @@
 import cgi
 import collections
 import contextlib
-import datetime
```

Comment:
I think it would be better to remove the import line 29 and keep the rest of the code: `date` doesn't look used in this file

Reply:
TBH I don't care which one we keep, as long as we don't import unused stuff like date here :)

Reply:
> in master `datetime` (the module) is unused, so it is better to keep `datetime` (the class)

But in 13 the classes are unused so removing just their import will only change one line (unless we also decide to fix the way-too-broad exception).

If we change the callsite then we might as well migrate it to `fields.Datetime.to_datetime` and remove both imports.

Reply:
in version 15, `datetime.now()` in this file. 

https://github.com/odoo/odoo/blob/15.0/odoo/http.py
```python
    def get_profiler_context_manager(self, request):
        """ Return a context manager that combines a profiler and ``request``. """
        if request.session.profile_session and request.session.db:
            if request.session.profile_expiration < str(datetime.now()):
```